test/test-perspective.el

```diff
@@ -259,4 +259,23 @@ persp-test-make-sample-environment."
     (should (not (get-buffer "*scratch* (A)")))
     (should (get-buffer "*scratch* (B)"))))
 
+(ert-deftest state-save-and-load ()
+  (unwind-protect
+      (persp-test-with-persp
+        (persp-test-with-files nil (A1 A2 A3 B1 B2 B3 B4)
+          (persp-test-make-sample-environment)
+          (should (= 7 (length (persp-test-buffer-list-all)))) ; sanity check
+          (persp-state-save "state-1.el"))
+        ;; reset perspectives
+        (persp-mode -1)
+        (delete-other-windows)
+        (should (= 0 (length (persp-test-buffer-list-all)))) ; no open files
```

Comment:
`seq-empty-p` seems clearer

Reply:
Yea. But AFAIK seq.el was only added in Emacs 25.1, and we're still running automated tests going back to 24.3.